scripts/generatePlatformSdk.sh

```diff
@@ -36,18 +81,34 @@ if [[ "${GENERATION_MODE}" != "docs" ]]; then
         --deprecatedFile "${SCRIPT_DIR}/../packages/deprecated/internal.foundry.core/core.json" \
         --endpointVersion "v1" \
         --mode "sdks" # We don't generate docs based on the OpenAPI IR
+  fi
+fi
+
+if [[ "${PREFIX}" == "all" || "${PREFIX}" == "foundry" ]]; then
+  echo "Generating foundry bindings"
+  $CODE_GENERATOR generate \
+      --v2 \
+      --prefix "foundry" \
+      --inputFile "${IR_JSON}" \
+      --manifestFile "${OPENAPI_MANIFEST_YML}" \
+      --outputDir "${OUT_PATH}" \
+      --deprecatedFile "${SCRIPT_DIR}/../packages/deprecated/foundry.core/core.json" \
+      --endpointVersion "v2" \
+      --mode "${GENERATION_MODE}"
 fi
 
-echo "Generating bindings"
-$CODE_GENERATOR generate \
-    --v2 \
-    --prefix "foundry" \
-    --inputFile "${IR_JSON}" \
-    --manifestFile "${OPENAPI_MANIFEST_YML}" \
-    --outputDir "${OUT_PATH}" \
-    --deprecatedFile "${SCRIPT_DIR}/../packages/deprecated/foundry.core/core.json" \
-    --endpointVersion "v2" \
-    --mode "${GENERATION_MODE}"
+if [[ "${PREFIX}" == "all" || "${PREFIX}" == "gotham" ]]; then
+  echo "Generating gotham bindings"
+  $CODE_GENERATOR generate \
+      --v2 \
+      --prefix "gotham" \
+      --inputFile "${IR_JSON}" \
+      --manifestFile "${OPENAPI_MANIFEST_YML}" \
+      --outputDir "${OUT_PATH}" \
+      --deprecatedFile "${SCRIPT_DIR}/../packages/deprecated/gotham.core/core.json" \
+      --endpointVersion "v2" \
+      --mode "${GENERATION_MODE}"
+fi
```

Comment:
you can probably dedupe this block and set the different values as variables depending on "${PREFIX}"